client/js/domhelpers.js

```diff
@@ -143,3 +143,7 @@ export function selectFile(getContents, multipleCallback) {
     upload.dispatchEvent(new MouseEvent('click', {bubbles: true}));
   });
 }
+
+export function asArray(variable) {
```

Comment:
We have toA(ids) in widgets.js doing similar and #777 is also adding a helper function to get a value as an array. Not really related to DOM, but it certainly is a useful helper function (all the other helper functions currently there are DOM related).

Reply:
This is replacing `toA`. I agree that it doesn't belong in that file but it's weird to introduce a new one with one tiny function.

Reply:
#777 is the exact same function with a different name. Those can just be merged when the time comes.

Reply:
I hadn't scrolled down enough to see that you'd touched widget.js in this commit.